frontend/common/src/components/AlertDialog/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import AlertDialogComponent from ".";
```

Comment:
Any reason not to import as `AlertDialog`? Seems that the story is implicit docs on how to use, so feels like a mixed msg to give a nonstandard story-specific name

Reply:
I've been messing around with implementation today to get it right before I put this up for review and I was changing names around and testing stuff out where I was using `AlertDialog` a lot so just used this name for now until I nail it down and create a proper story.